bob/src/cluster/quorum.rs

```diff
@@ -26,6 +26,7 @@ impl Quorum {
     }
 
     async fn put_at_least(&self, key: BobKey, data: BobData) -> Result<(), Error> {
```

Comment:
Why not by ref?

Reply:
This is the place where data is consumed, so ownership is transfered here